modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -100,6 +102,23 @@
 				)
 			);
 		},
+		syncByReachedCuePoints : function()
```

Comment:
This invoke a sync by cue points. If no sync was done it indicate that we should not show any image